### DISCUSSION TITLE
CLS and Pagespeed Score on Mobiles

### DISCUSSION THREAD

**Discussion by philManWithBeard** [incluud/accessible-astro-starter#72] [General]
Hi there, I've noticed that Pagespeed scores for mobile are lower than expected because of CLS. I was wondering whether I could help to improve this? Let me know if you've already got ideas in mind or I can start contributing ones here.

**Comment by markteekman** [comment]
Hey @philManWithBeard! Thanks for your suggestion. I'm always open to help from the community, especially since I've been short on time for my Astro projects the last couple of months. So feel free to make contributions to improve pagespeed scores for mobile, that sounds like a big win to me! 😃
